control-plane/api-gateway/controllers/index.go

```diff
@@ -336,9 +338,18 @@ func gatewayForGatewayPolicy(o client.Object) []string {
 	gatewayPolicy := o.(*v1alpha1.GatewayPolicy)
 
 	targetGateway := gatewayPolicy.Spec.TargetRef
-	//gateway policy is 1to1
-	if targetGateway.Group == v1alpha1.ConsulHashicorpGroup && targetGateway.Kind == common.KindGateway {
-		return []string{common.IndexedNamespacedNameWithDefault(targetGateway.Name, &targetGateway.Namespace, gatewayPolicy.Namespace).String()}
+	// gateway policy is 1to1
+	fmt.Println()
+	if targetGateway.Group == "gateway.networking.k8s.io/v1beta1" && targetGateway.Kind == common.KindGateway {
```

Comment:
mad some changes here because this was not working as expected

Reply:
Let's add `"gateway.networking.k8s.io/v1beta1"` to common/constants.go so we can change it more easily when we upgrade in the future.

Reply:
good catch! this constant exists in the k8s sig package, I thought I caught all the refs I had hardcoded (this will also probably change in @sarahalsmiller PR where she's implementing it